user_tools/docs/index.md

```diff
@@ -132,6 +132,25 @@ The following table summarizes the commands supported for each cloud platform:
 |                  |               +-------------+---------------------------------------+----------+
 |                  |               | cluster     |           unsupported                 |    N/A   |
 +------------------+---------------+-------------+---------------------------------------+----------+
+| OnPrem           | qualification | local       | spark_rapids_user_tools \             |  23.06+  |
+|                  |               |             |   onprem qualification [ARGS]         |          |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | serverless  |           unsupported                 |    N/A   |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | cluster     |           unsupported                 |    N/A   |
+|                  +---------------+-------------+---------------------------------------+----------+
+|                  | profiling     | local       |           unsupported                 |    N/A   |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | serverless  |           unsupported                 |    N/A   |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | cluster     |           unsupported                 |    N/A   |
+|                  +---------------+-------------+---------------------------------------+----------+
+|                  | bootstrap     | local       |           unsupported                 |    N/A   |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | serverless  |           unsupported                 |    N/A   |
+|                  |               +-------------+---------------------------------------+----------+
+|                  |               | cluster     |           unsupported                 |    N/A   |
++------------------+---------------+-------------+---------------------------------------+----------+
```

Comment:
We may need to update the graph as well because the diagram does not show `onprem`.
Probably I should add the source of the overview.draw file to resources.

Reply:
I created a PR adding the source file of the overview
https://github.com/NVIDIA/spark-rapids-tools/pull/304